app/code/Magebit/PageListWidget/etc/module.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
```

Comment:
Dock types in XML as well:
![image](https://user-images.githubusercontent.com/69341511/221507700-0c17c43e-4942-478b-8dad-86f93036aaf5.png)